signer/migrations/0003__create_tables.sql

```diff
@@ -184,3 +191,111 @@ CREATE TABLE sbtc_signer.withdrawal_reject_events (
     created_at    TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP NOT NULL
 );
 
+-- Represents a combined transaction package which is broadcasted to the Bitcoin
+-- network. The transaction package is built up of multiple transactions which
+-- are tracked separately in the `sweep_transactions` table. A transaction
+-- package may contain the sweeping transactions for both deposit and
+-- withdrawal requests.
+CREATE TABLE sbtc_signer.sweep_packages (
+    -- Internal ID of the package
+    id SERIAL PRIMARY KEY,
+    -- The Bitcoin block hash at which this package was created.
+    created_at_block_hash BYTEA NOT NULL,
+    -- The Bitcoin market fee rate at the time this package was created.
+    market_fee_rate DOUBLE PRECISION NOT NULL,
+    -- Timestamp of when this package was created.
+    created_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP NOT NULL
+);
+
+-- Represents an individual transaction within a broadcasted sweep transaction
+-- package. Individual deposit and withdrawal requests reference back to these
+-- transactions to keep track of both the overall transaction package as well as
+-- the individual Bitcoin transactions they are related to.
+CREATE TABLE sbtc_signer.sweep_transactions (
+    -- Internal ID of the transaction.
+    id BIGSERIAL PRIMARY KEY,
+    -- The ID of the grouping transaction package.
+    sweep_package_id INTEGER NOT NULL,
+    -- The Bitcoin transaction ID of the transaction.
+    txid BYTEA NOT NULL,
+    -- The signer UTXO being spent in this transaction.
+    utxo_txid BYTEA NOT NULL,
+    utxo_output_index INTEGER NOT NULL,
```

Comment:
Hmmm, I think we want `scriptPubKey` and `amount` columns here as well.

Reply:
nit: we might want to rename this to `signer_prevout_txid` and `signer_prevout_output_index` or something similar. The `utxo_` prefix is ambiguous on it's own, you need to know that you are referring to the signers' UTXO.

Reply:
Renamed and added `signer_prevout_amount` and `signer_prevout_script_pubkey` in [2ee116d](https://github.com/stacks-network/sbtc/pull/710/commits/2ee116d31948a4e5f2f5e95512b468aa5ecf8cb2). I'm not sure I'm putting the actual expected value in the script pubkey column, is this correct?
```rust
transaction
  .signer_utxo
  .utxo
  .public_key
  .signers_script_pubkey()
  .into(),
```

Reply:
Yeah, that is correct.

Reply:
Awesome :pray:

Reply:
@djordon can this comment be closed?

Reply:
yup!